src/NUnitFramework/tests/Assertions/CollectionAssertTest.cs

```diff
@@ -563,9 +565,9 @@ public void DoesNotContain_Fails()
             var ex = Assert.Throws<AssertionException>(() => CollectionAssert.DoesNotContain(list,"y"));
             Assert.That(ex.Message, Is.EqualTo(expectedMessage));
         }
-        #endregion
+#endregion
 
-        #region IsSubsetOf
+#region IsSubsetOf
```

Comment:
I really wish Visual Studio would fix this formatting issue. Makes me want to do a mass unindent checkin.

Reply:
I would love it if you did. That, and an endianness fix, and a trailing whitespace fix. Or just say it's okay for me to. 😀